Gemfile

```diff
@@ -1,3 +1,4 @@
 source "http://rubygems.org"
+gem 'activesupport', '~> 3.2.22'
```

Comment:
Why is this needed? Pretty heavy weight dependency to pull in for a relatively small gem, also the rails version lock is going to be challenging.  In any case, gems served by rubygems should be included as a dependency in in the gemspec so bundler can parse dependencies.